examples/interaction/call-activity-with-tabs-on-click/index.js

```diff
@@ -0,0 +1,389 @@
+// Inspired from 'Procurement Processes with Error Handling - Stencil Trisotech 3 pages.bpmn' of https://www.omg.org/cgi-bin/doc?dtc/10-06-02.zip
+// Duplicated from examples/interaction/call_activity_with_modal_on_mouse_over
+function getMainBpmnDiagram() {
```

Comment:
:question: can we introduce an utils script file that will store the bpmn content function to let us reuse the functions in the call activity related examples?
In addition to remove duplications, this will make this script simpler

Reply:
Yes, I think I'll do in another PR 😉

Reply:
ok,no problem